products/views.py

```diff
@@ -1,3 +1,33 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http           import JsonResponse
+from django.views          import View
+
+from products.models  import Product, ProductImage, TasteByProduct
+
+
+class MainProductView(View): 
+    def get(self, request): 
+        premiums             = Product.objects.all().order_by('-price')[:3]
+        fresh_products       = Product.objects.all().order_by('-roasting_date')[:4]
+        result_premium       = [{
+                    'id'            : premium.id,
+                    'name'          : premium.name,
+                    'eng_name'      : premium.eng_name,
+                    'img'           : [image.url for image in ProductImage.objects.filter(product_id = premium.id)],
```

Comment:
```suggestion
                    'images'        : [{
                        'id'  : image.id,
                        'url' : image.url
                        } for image in ProductImage.objects.filter(product_id = premium.id)],
```
적용 잘해주셨네요! 좋습니다 상현님 👍 

다만 client (frontend)에게 데이터를 Return 할 때에는 id(pk)값을 같이 반환해주는 것이 좋습니다.

다른 곳에도 반영하여 수정해주세요~